.github/actions/update-documentation/action.yml

```
@@ -0,0 +1,88 @@
+name: "Update Keptn Lifecycle Toolkit Documentation and Examples"
+description: "Update Keptn Lifecycle Toolkit Documentation and Examples"
+inputs:
+   version:
+     required: true
+     description: "Version of the Keptn Lifecycle Toolkit Documentation to be deployed"
+   doc-repo:
+     required: true
+     description: "Path to the documentation repository"
+     default: "keptn-sandbox/lifecycle-toolkit-docs"
+   examples-repo:
+     required: true
+     description: "Path to the examples repository"
+     default: "keptn-sandbox/lifecycle-toolkit-examples"
+   klt-repo:
+     required: true
+     description: "Path to the klt repository"
+     default: "lifecycle-toolkit"
+   update-main:
+     description: "Update the main version of the documentation"
+     required: true
+     default: "false"
+   target-branch:
+     description: "Target branch for the documentation"
+     default: "next"
+     required: true
+   token:
+     description: "Token to access the documentation repository"
+     required: true
+
+
+runs:
+  using: "composite"
+
+  steps:
+  - uses: actions/setup-python@v4
+    with:
+      python-version: '3.10'
+
+  - name: Check out documentation
+    uses: actions/checkout@v3
+    with:
+      repository: ${{ inputs.doc-repo }}
+      path: doc-repo
+      ref: main
+      token: ${{ inputs.token }}
+      fetch-depth: 0
+
+  - name: Check out examples
+    uses: actions/checkout@v3
+    if: ${{ inputs.update-main == 'true' }}
```

you don't need the curly braces in if statements
```suggestion
    if: inputs.update-main == 'true'
```